packages/x-charts-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-pro",
-  "version": "7.0.0-beta.7",
+  "version": "7.22.3",
```

Comment:
Would it make sense to release the base charts package to have the versions in sync and the version selector in the header represent correct data? 🤔